docs/python.md

```diff
@@ -0,0 +1,60 @@
+<!--ts-->
+   * [Using Bazel](#using-bazel)
+   * [Manual Setup Illustration](#manual-setup-illustration)
+
+<!-- Added by: advaitjain, at: Fri Oct 29 11:57:14 AM PDT 2021 -->
+
+<!--te-->
+
+Writing and using Python scripts from the TFLM repository is currently in the
+prototyping stage. As such, the instrucitons below are somewhat sparse and
+subject to change.
+
+
+* [TensorFlow Python style guide](https://www.tensorflow.org/community/contribute/code_style#python_style)
+
+
+# Using Bazel
+
+We use Bazel as our defaul build system for both Python and C++ and the
+continuous integration system uses bazel to run the Python unit tests.
```

Comment:
nit: just "python"?  I feel mentioning python and c++ together in this phrase together may lead people to infer c++ also use bazel as default, but I think for C++, make is at least as important as bazel. WDYT?

Reply:
good point. changed to mention Python only.